src/Microsoft.Data.SqlClient/src/Microsoft/Data/Common/DbBatch.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Data;
+using System.Data.Common;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Data.Common
+{
+    /// <include file='../../../../doc/snippets/Microsoft.Data.Common/DbBatch.xml' path='docs/members[@name="DbBatch"]/DbBatch/*' />
+    public abstract partial class DbBatch : IDisposable
```

Comment:
Same as previous comments, I think this becomes internal-only for netfx, right?

Reply:
see above